tasks/dataset-split/utils.py

```diff
@@ -0,0 +1,18 @@
+from typing import TypeVar
+
+from coretex import NetworkDataset, NetworkSample
+
+SampleType = TypeVar("SampleType", bound = NetworkSample)
+DatasetType = TypeVar("DatasetType", bound = NetworkDataset)
+
+
+def samplesSplit(originalDataset: DatasetType, datasetCount: int) -> list[list[SampleType]]:
```

Comment:
`splitSamples`. It is always a better idea to name the functions in a way which describes what they do